app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java

```diff
@@ -65,6 +65,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     void initBottomNavigation() {
         viewPager.setAdapter(new BottomNavigationFragmentPagerAdapter(getSupportFragmentManager()));
+        viewPager.setOffscreenPageLimit(0);
```

Comment:
I think you cant do that.
`Requested offscreen page limit 0 too small; defaulting to 1`